langchain4j-cohere/src/main/java/dev/langchain4j/model/cohere/ChatHistory.java

```diff
@@ -0,0 +1,16 @@
+package dev.langchain4j.model.cohere;
+
+
+import lombok.Builder;
+import lombok.NonNull;
+
+@Builder
+public class ChatHistory {
```

Comment:
Minor: we are now changing the way classes are organized in the modules. Could you please use the [following](https://github.com/langchain4j/langchain4j/tree/main/langchain4j-anthropic/src/main/java/dev/langchain4j/model/anthropic) structure (inside the `internal` package)?